packages/material-ui-system/src/index.d.ts

```diff
@@ -56,7 +56,7 @@ export function compose<T extends Array<StyleFunction<any>>>(...args: T): Compos
 // css.js
 export function css<Props>(
   styleFunction: StyleFunction<Props>
-): StyleFunction<Props & { css: Omit<Props, 'theme'> }>;
+): StyleFunction<Props & { css?: Omit<Props, 'theme'> }>;
```

Comment:
What about making a different pull request for the css concern? It would also be a good opportunity to add a JavaScript test case for it, I wasn't aware it was even working. Actually, I don't think that we should support it, rather encourage the other way around